scripts/packages/BUILD

```diff
@@ -474,9 +474,11 @@ genrule(
         'rm setup.py.template',
         'tree $$HERONPY_DIR',
         '/usr/bin/env python2.7 setup.py sdist',
-        '/usr/bin/env python2.7 setup.py bdist_wheel',
+        '/usr/bin/env python2.7 setup.py bdist_wheel --universal',
+        'cd -',
         'ls -l $$HERONPY_DIR/dist',
-        'cp $$HERONPY_DIR/dist/heronpy-*-py2-*.whl $$OUTPUT_DIR',
+        'cp $$HERONPY_DIR/dist/heronpy-*-py2.py3-*.whl $$OUTPUT_DIR',
+        'cp $$HERONPY_DIR/dist/heronpy-*-py2.py3-*.whl "$@"',
```

Comment:
This wasn't being done although it is supposed to be a product of this rule so I added it.

Reply:
@Code0x58 - wondering if you had a chance to test this with py3 topologies?

Reply:
I haven't quite got to running any PEXs yet, but should have done so by the end of this weekend

Reply:
The universal package works locally, so the next step is to make sure it can run on the executor image. I'll get stuck into changing the Dockerfiles to include python3.6 over the next few days. My thinking is that they should include the latest stable python, so PEXs only need to have python3 in their shebang and they'll just work.

_p.s. the WIP commit is ugly, I pushed it to switch machines for a few days_